README.md

```diff
@@ -105,8 +105,8 @@ avoid duplication of effort.
 Despite some differences, the Go implementation of Starlark strives to
 match the behavior of the Java implementation used by Bazel.
 For that reason, proposals to change the language itself should
-generally be directed to the Bazel team, not to the maintainers of
-this project.
+generally be directed to [the Bazel team](https://github.com/bazelbuild/bazel/issues),
+not to the maintainers of this project.
```

Comment:
We have a separate repositories that fits better: https://github.com/bazelbuild/starlark/

(I recommend using https://github.com/bazelbuild/bazel only for bugs in the Bazel implementation of Starlark)

Reply:
I can send a new pull request for that